server/src/TUManager.cpp

```diff
@@ -48,9 +48,15 @@ TUManager::TUManager()
   // significantly slower.
   //
   // -- https://github.com/Sarcasm/irony-mode/issues/4
-  if (CINDEX_VERSION < 6) {
-    parseTUOptions_ &= ~CXTranslationUnit_PrecompiledPreamble;
-  }
+#if CINDEX_VERSION < 6
```

Comment:
Thanks for the clarification.

In the diagnostic code, the preprocessor check was used, so I thought that was the convention and changed the one place I found it to be different.  I'll leave it alone.